pulp_rpm/app/tasks/publishing.py

```diff
@@ -103,10 +103,21 @@ def publish_artifacts(self, content):
         """
         published_artifacts = []
         for content_artifact in ContentArtifact.objects.filter(
-                content__in=content.exclude(pulp_type="rpm.repo_metadata_file").distinct()
+                content__in=content.exclude(
+                    pulp_type="rpm.repo_metadata_file"
+                ).distinct()
         ).iterator():
+            relative_path = (
+                os.path.join(
+                    "Packages",
+                    content_artifact.relative_path[0].lower(),
```

Comment:
So what do we have in the relative path? only filename? always? Is it safe to do `content_artifact.relative_path[0]` and assume that it's the first letter of the filename?

@ppicka, @ipanova ,  @dralley , @daviddavis

Reply:
@goosemania yes we have only filename in the relative_path` filename = os.path.basename(package.location_href)` This is the code from sync that confirms that.